code/parachain/runtime/composable/src/fees.rs

```diff
@@ -67,6 +67,8 @@ impl asset_tx_payment::HandleCredit<AccountId, Tokens> for TransferToTreasuryOrD
 
 parameter_types! {
 	pub AssetConfigHoldIdentifier: TemporalHoldIdentifier = ();
+	pub NativeAssetId: CurrencyId = CurrencyId::PICA;
+	pub DefaultFeeAsset: CurrencyId = CurrencyId::DOT;
```

Comment:
just to be sure DOT id is corrent, it is dot on Composable and will work right.

Reply:
you are correct, this isnt composable Dot